meshtastic/interfaces/ble/notifications.py

```
@@ -228,3 +283,464 @@ def _get_callback(self, characteristic: str) -> Callable[[Any, Any], None] | Non
+        ):
+            hook = getattr(error_handler, hook_name, None)
+            if callable(hook) and not _is_unconfigured_mock_callable(hook):
+                report_exception = hook
+                break
+        if report_exception is not None:
+            try:
+                report_exception(error_msg)
+            except Exception:  # noqa: BLE001 - callback error reporting is best effort
+                logger.debug(error_msg, exc_info=True)
+            return
+        logger.debug(error_msg, exc_info=True)
```

Ruff (0.15.6)

[warning] `exc_info=` outside exception handlers

Remove `exc_info=`

(LOG014)